src/components/menu-bar/login-dropdown.jsx

```diff
@@ -7,11 +7,38 @@ eventually be consolidated.
 import classNames from 'classnames';
 import PropTypes from 'prop-types';
 import React from 'react';
+import {defineMessages} from 'react-intl';
 
 import MenuBarMenu from './menu-bar-menu.jsx';
 
 import styles from './login-dropdown.css';
 
+// these are here as a hack to get them translated, so that equivalent messages will be translated
+// when passed in from www via gui's renderLogin() function
+const LoginDropdownMessages = defineMessages({ // eslint-disable-line no-unused-vars
+    backdrop: {
```

Comment:
keys should be something related to the message, e.g., `username:`

Reply:
Sorry, totally missed these! Updated!